pyseir/inference/model_fitter.py

```diff
@@ -99,7 +106,16 @@ class ModelFitter:
         errordef=0.5,
     )
 
-    PARAM_SETS = {("HI",): dict(eps=0.25, t0=75, t_break=10, limit_t0=[50, 90])}
+    PARAM_SETS = {
+        ("HI"): dict(eps=0.25, t0=75, t_break=10, limit_t0=[50, 90]),
+        ("AK"): dict(eps=0.25, t0=75, t_break=10, limit_t0=[50, 90]),
+        ("MT"): dict(eps=0.25, t0=75, t_break=10, limit_t0=[50, 90]),
+        ("ID"): dict(eps=0.25, t0=75, t_break=10, limit_t0=[50, 90]),
+        ("LA"): dict(eps=0.25, t0=75, t_break=10, limit_t0=[50, 90]),
+        ("ND"): dict(eps=0.25, t0=75, t_break=10, limit_t0=[50, 90]),
+        ("WV"): dict(eps=0.25, t0=75, t_break=10, limit_t0=[50, 90]),
+        ("WY"): dict(eps=0.25, t0=75, t_break=10, limit_t0=[50, 90]),
```

Comment:
Since the dict being used is the same across all of these states, I'd pull it out into a variable to avoid the duplication.

Reply:
Good point, but I am not sure we won't want to update these. So let's make that a future decision : )

Reply:
fixed